src/NzbDrone.Core/Indexers/Definitions/Rarbg/Rarbg.cs

```diff
@@ -16,7 +16,15 @@ public class Rarbg : TorrentIndexerBase<RarbgSettings>
         private readonly IRarbgTokenProvider _tokenProvider;
 
         public override string Name => "Rarbg";
-        public override string[] IndexerUrls => new string[] { "https://torrentapi.org" };
+        public override string[] IndexerUrls => new string[] 
+        { 
+          "https://torrentapi.org",
+          "https://rarbg.to", 
```

Comment:
I'm not sure that all of these actually work as API alternatives. They are more site/info url alternatives. The code here needs a few more adjustments to know when to use what

Reply:
Simple adjustments or something a bit more complex?

Reply:
Should be simple enough

Reply:
Q I'll leave the PR up, but defer to you or someone else where the specific changes are to be made and/or just pushing those up.

unfortunately don't have time this/next week for anything much more than quick easy commits